client/src/connection/index.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright © 2022, SAS Institute Inc., Cary, NC, USA. All Rights Reserved.
```

Comment:
Should this be Copyright 2023? Same comment for the others

Reply:
And, if we update these to 2023, should we go ahead and do it for all the others?

Reply:
Thank you for pointing it out. I added 2022 because the file was originally authored in 2022.  I checked our rule. It should be updated to be a range when modified in a new year. But if a file is not modified, we don't need to update it. I've go through all files and updated them acoordingly.